packages/protocol-utils/src/transformer_utils.ts

```diff
@@ -52,19 +80,91 @@ export enum FillQuoteTransformerSide {
     Buy,
 }
 
+/**
+ * Represents either `FillQuoteTransformer.LimitOrderInfo`
+ * or `FillQuoteTransformer.RfqOrderInfo`
+ */
+interface FillQouteTransformerNativeOrderInfo<T> {
+    order: T;
+    signature: Signature;
+    maxTakerTokenFillAmount: BigNumber;
+}
+
+/**
+ * `FillQuoteTransformer.OrderType`
+ */
+export enum FillQuoteTransformerOrderType {
+    Bridge,
+    Limit,
+    Rfq,
+}
+
 /**
  * `FillQuoteTransformer.TransformData`
  */
 export interface FillQuoteTransformerData {
     side: FillQuoteTransformerSide;
     sellToken: string;
     buyToken: string;
-    orders: Array<Exclude<Order, ['signature', 'exchangeAddress', 'chainId']>>;
-    signatures: string[];
-    maxOrderFillAmounts: BigNumber[];
+    bridgeOrders: FillQuoteTransformerBridgeOrder[];
+    limitOrders: Array<FillQouteTransformerNativeOrderInfo<LimitOrder>>;
+    rfqOrders: Array<FillQouteTransformerNativeOrderInfo<RfqOrder>>;
+    fillSequence: FillQuoteTransformerOrderType[];
     fillAmount: BigNumber;
     refundReceiver: string;
-    rfqtTakerAddress: string;
+}
+
+/**
+ * Identifies the DEX type of a bridge order.
+ */
+export enum BridgeSource {
```

Comment:
Should comment that this should be append only, or we explicitly set the value. One might assume a new source should be added alphabetically